src/Apps/Auction/getRedirect.ts

```diff
@@ -0,0 +1,87 @@
+import { routes_BidQueryResponse } from "__generated__/routes_BidQuery.graphql"
+import { routes_RegisterQueryResponse } from "__generated__/routes_RegisterQuery.graphql"
+
+export interface Redirect {
+  path: string
+  reason: string
+}
+
+export function registerRedirect(
+  sale: routes_RegisterQueryResponse["sale"],
+  me: routes_RegisterQueryResponse["me"]
```

Comment:
It's a little friendlier to use objects as arguments, similar to React's `props` pattern. That way things aren't order dependent, while also being easily introspectable by logging the single arg and seeing object keys.

Reply:
Agreed, & i think this may change eventually as I work towards consolidating this redirect logic elsewhere. It only has one callsite right now so going to keep as is.